app/models.py

```diff
@@ -0,0 +1,230 @@
+from peewee import *
+
+DATABASE_NAME = "database.db"
+database = SqliteDatabase(DATABASE_NAME)
+
+
+# Database models
+class BaseModel(Model):
+    class Meta:
+        database = database
+
+
+class User(BaseModel):
+    username = CharField(unique=True)
+    fullname = CharField()
+    mail_address = CharField()
+    password = CharField()
+    user_type = CharField()
+
+    def is_student(self):
+        return User.user_type == "is_student"
```

Comment:
How about create a model `UserType` for this with default values?

Reply:
Good idea! I wasn't sure how to do it though, won't a named tuple suffice?

Reply:
I'd task this a step further, and a `UserType` that can be configured for permissions, ie:
Teacher ->
   Courses: Read Write
   Grades: Read Write
Student ->
    Courses: Read
   Exercises: Read Write